hw4/02-sessions.js

```diff
@@ -1,17 +1,38 @@
+
 const express = require('express');
 const session = require('express-session');
-const app = express();
-const port = process.env.PORT || 5001;
+const app     = express();
+const port    = process.env.PORT || 5001;
+
 
-// Add your code here
+app.use(session({secret: 'secret_key'}));
 
-// Use the express-session module
-app.use(/** ... */);
 
 app.get('/', (req, res) => {
-  // Add your code here
+  if (req.session.first_time) {
+    req.session.routes.push('/');
+    res.send(`Navigate to any route.`);
+  }
+  else {
+    req.session.first_time = true;
+    req.session.routes = ['/'];
```

Comment:
Even this part needs to be changed as the user could enter the website to any type of URL and there needs to be a record of that visit.